cla_frontend/apps/zendesk/client.py

```diff
@@ -0,0 +1,81 @@
+import json
+import requests
+from requests.exceptions import ConnectionError, Timeout
+from django.conf import settings
+from django.template import loader, Context
+
+TICKETS_URL = 'https://ministryofjustice.zendesk.com/api/v2/tickets.json'
+ZENDESK_REQUESTER_ID = 762871298 # Civil Legal Advice
+ZENDESK_GROUP_ID = 24287107 # CLA Operator/Provider
+REQUEST_TIMEOUT = 10
+
+
+class ZendeskResponse(object):
+    def __init__(self, **kwargs):
+        self.__dict__.update(kwargs)
+
+
+class ZendeskClient(object):
+
+    def feedback_payload(self, feedback_type, data):
+        t = loader.get_template('zendesk-feedback.txt')
+        feedback_body = t.render(Context(data))
+
+        custom_fields = []
+        tags = ['feedback', feedback_type]
+
+        if data['app_name']:
```

Comment:
If these keys must be in the `data` dict, then these tests are redundant - just access `data['app_name']`